src/core/public/chrome/nav_links/nav_links_service.ts

```diff
@@ -151,6 +164,21 @@ export class NavLinksService {
         return navLinks$.pipe(map(sortNavLinks), takeUntil(this.stop$));
       },
 
+      setFilteredNavLinks: (filteredNavLinks: ReadonlyMap<string, ChromeNavLink>) => {
+        this.filteredNavLinks$.next(filteredNavLinks);
+      },
+
+      getFilteredNavLinks$: () => {
```

Comment:
Since `getNavLinks$` is only used in workspace plugin.ts, how about renaming `getNavLinks$` to `getAllNavLinks$` and `getFilteredNavLinks$` to `getNavLinks$`?

Reply:
That's make sense to me.